os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -87,6 +87,10 @@ config AMEBASMART_I2C
 	bool "AMEBASMART_I2C"
 	default n
 
+config AMEBASMART_MIPI
+	bool "AMEBASMART_MIPI"
+	default n
```

Comment:
Code is ok but how about adjusting the location?
```
config I2C
   <--- You added here
if I2C
```

You add the MIPI config at middle of I2C config. How about moving it before or after I2C?